translator/trace/jaeger/protospan_to_jaegerproto_test.go

```diff
@@ -15,8 +15,8 @@
 package jaeger
 
 import (
-	"encoding/json"
 	"fmt"
+	"github.com/google/go-cmp/cmp"
```

Comment:
Style: our convention for import ordering is to have 3 sections:
1. Standard packages
2. 3rd party packages
3. Our own packages
Each section sorted alphabetically and separated by a blank line from the other sections.

Reply:
Will be fixed in the next pr pushed.